dev/create-release/release-build.sh

```diff
@@ -225,9 +225,9 @@ if [[ "$1" == "publish-snapshot" ]]; then
 
   $MVN -DzincPort=$ZINC_PORT --settings $tmp_settings -DskipTests $PUBLISH_PROFILES \
     -Phive-thriftserver deploy
-  ./dev/change-scala-version.sh 2.10
+  ./dev/change-scala-version.sh 2.11
```

Comment:
Ah sorry just looked at this the right way around -- should the snapshot target 2.11 or 2.10? I would have thought 2.10, but, either could be reasonable